lib/index.js

```diff
@@ -0,0 +1,143 @@
+async function goHome() {
+  await browser.goTo('#Shell-home');
+  await browser.waitForUI5();
+}
+
+async function pressTile(name) {
+  const control = await browser.asControl({
+    selector: {
+      controlType: "sap.m.GenericTile",
+      properties: {
+        header: name
+      }
+    }
+  });
+  await control.press()
+}
+
+async function searchFor(text) {
+  await browser.asControl({
+    selector: {
+      controlType: "sap.m.SearchField",
+      interaction: {
+        idSuffix: "I"
+      }
+    }
+  }).enterText(text)
+}
+
+async function performStandardAction( name ) {  
```

Comment:
whitespacing btw method and parenthesis → please introduce a formatter (e.g. `prettier`) and optionally a linter (e.g. `eslint`) for avoiding style issues.
I'd suggest to align with `wdi5` with that: https://github.com/ui5-community/wdi5/blob/main/.prettierrc

Reply:
added prettier as suggested